spec/controllers/games_controller_spec.rb

```diff
@@ -2,4 +2,29 @@
 
 RSpec.describe GamesController, type: :controller do
 
+  describe 'games#create action' do
+
+    it "should successfully create a game in the database" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+
+      post :create, game: {name: 'Testgame'}
+
+      game = Game.last
+      expect(game.name).to eq("Testgame")
+    end
+
+    it "should add the current user as white player when creating game" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+
+      post :create, game: {name: 'Testgame',
+                            player_white_id: user.id}
```

Comment:
What happens if you don't specify 'player_white_id'. This is the exact thing you want to test that when creating a game the current user is automatically the white player..